README.md

```diff
@@ -1,45 +1,31 @@
 # Dataspace Protocol
 
-## About versions of the Dataspace Protocol
+The __Dataspace Protocol__ is a set of specifications designed to facilitate interoperable data sharing between entities governed by usage control and based on Web technologies. These specifications define the schemas and protocols required for entities to publish data, negotiate [Agreements](https://eclipse-dataspace-protocol-base.github.io/DataspaceProtocol/#dfn-agreement), and access data as part of a federation of technical systems termed a [Dataspace](https://eclipse-dataspace-protocol-base.github.io/DataspaceProtocol/#dfn-dataspace).
 
-Since [version 0.8](https://github.com/eclipse-dataspace-protocol-base/DataspaceProtocol/tree/main/releases/v0.8)
-the specification has been stable with changes in details. Its The last release candidate ([2024-1](https://docs.internationaldataspaces.org/ids-knowledgebase/dataspace-protocol))
-of the Dataspace Protocol specification is considered to be stable. It was specified under governance of the [International Dataspaces Association](https://internationaldataspaces.org/). 
-Further changes shall not affect conformity. 
+The web rendering of the spec represents the current state on this repo's main branch:
+https://eclipse-dataspace-protocol-base.github.io/DataspaceProtocol/.
 
-Now, the spec is maintained and developed by the [`eclipse-dataspace-protocol-base`](https://projects.eclipse.org/projects/technology.dataspace-protocol-base) 
-Eclipse Project under the umbrella of the [Eclipse Dataspace Working Group](https://dataspace.eclipse.org/) (EDWG).
-The source of truth is this repository. 
+## About versions
 
-The web rendering of the spec represents the current state on this repo's main branch.
-https://eclipse-dataspace-protocol-base.github.io/DataspaceProtocol/
+Since [Version 0.8](https://github.com/eclipse-dataspace-protocol-base/DataspaceProtocol/tree/main/releases/v0.8), the specification has been stable with changes in details. The latest release candidate ([2024-1](https://docs.internationaldataspaces.org/ids-knowledgebase/dataspace-protocol)) of the Dataspace Protocol specification is considered stable. It has been developed under the governance of the [International Data Spaces Association](https://internationaldataspaces.org/). Further changes shall not affect conformity. 
 
-> __NOTE:__ A versioning scheme beside the commits to the repository is not available but will be provided in the
-> future.
+Now, the spec is maintained and developed by the [Eclipse Dataspace Protocol](https://projects.eclipse.org/projects/technology.dataspace-protocol-base) project under the purview of the [Eclipse Dataspace Working Group](https://dataspace.eclipse.org/) (EDWG). The source of truth is this repository. 
```

Comment:
```suggestion
The spec is maintained and developed by the [Eclipse Dataspace Protocol](https://projects.eclipse.org/projects/technology.dataspace-protocol-base) project under the purview of the [Eclipse Dataspace Working Group](https://dataspace.eclipse.org/) (EDWG). 
```

Reply:
Please change to [Eclipse Dataspace Protocol Project], as it is a project and the result is still the Dataspace Protocol.